src/BinSkim.Rules/PERules/BA2004.EnableSecureSourceCodeHashing.cs

```diff
@@ -164,6 +165,12 @@ public void AnalyzeNativeBinaryAndPdb(BinaryAnalyzerContext context)
                             {
                                 continue;
                             }
+                            else if (sf.FileName.EndsWith(".winmd"))
```

Comment:
>else if (sf.FileName.EndsWith(".winmd")) [](http://example.com/codeflow?start=28&length=40)

may consider add how the test file `WindowsRuntimeComponentTest-VS2022.dll`
created with what template used in
`docs\FunctionalTestBuildScripts.md`